npc/re/custom/lasagna/lasagna_npcs.txt

```diff
@@ -1435,6 +1439,11 @@ izlude,138,183,3	script	Vigilante Penne#doect01	4_DR_SOLDIER,{
 	}
 }
 
+izlude_a,138,183,3	duplicate(Vigilante Penne#doect01)	Vigilante Penne#doect01_a	4_M_MERCAT1
+izlude_b,138,183,3	duplicate(Vigilante Penne#doect01)	Vigilante Penne#doect01_b	4_M_MERCAT1
+izlude_c,138,183,3	duplicate(Vigilante Penne#doect01)	Vigilante Penne#doect01_c	4_M_MERCAT1
+izlude_d,138,183,3	duplicate(Vigilante Penne#doect01)	Vigilante Penne#doect01_d	4_M_MERCAT1
```

Comment:
They didn't duplicated Vigilante on kRO (thanks @Lemongrass3110 for checking), please remove theses 4 duplicates